### DISCUSSION TITLE
Build process keeps getting killed in Docker

### DISCUSSION THREAD

**Discussion by fnick851** [vercel/next.js#45626] [Help]
And it kills the process without showing a reason.

<img width="388" alt="Screenshot 2023-02-06 at 12 18 36" src="https://user-images.githubusercontent.com/8741707/217040181-d07e317c-5f6e-431f-9921-5329017c01b6.png">
<img width="684" alt="Screenshot 2023-02-06 at 12 18 53" src="https://user-images.githubusercontent.com/8741707/217040191-e7815b9c-ee79-4566-8167-acfd6103f807.png">

I do notice the CPU usage is high but that's normal I think for Node.

Does anyone has some ideas - what might cause this?
I increased memory and CPU for the container so I don't think it is running out of memory or CPU. 
It might be some Linux configuration?

Thanks!

**Comment by DanLeininger** [comment]
@fnick851 I'm running into this too, did you ever figure out what was killing the build?

**Reply by fnick851** [reply]
So this was in a Docker container. And we eventually think the reason is still memory. We increased the resources for the container and it seems like the issue is gone now. 

I suggest experimenting with allocating more resources to the environment you use. This is likely not an issue with the framework itself.

**Comment by d-cs** [comment]
I'm getting this when running on M2 processor trying to build a linux/amd64 container

**Comment by bloodykheeng** [comment]
try to increase your swap memory
for me my issue was that i couldnt npm run build on linux server
out of the blue the terminal was giving me message killed

after some research i found out that my swap memory was 500mb and i extended it to 1gb so the build ran well now
my server was 4gb

to check for swap
cd into root
then type

```
free
```
that will show u the available swap memory

the commands below will increase it to 1gb

```
  sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=1024
  sudo /sbin/mkswap /var/swap.1
  sudo /sbin/swapon /var/swap.1
```
